chart/Chart.yaml

```diff
@@ -2,21 +2,21 @@ annotations:
   catalog.cattle.io/auto-install: rancher-cis-benchmark-crd=match
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: CIS Benchmark
-  catalog.cattle.io/kube-version: '>= 1.27.0-0 < 1.31.0-0'
```

Comment:
Would this not start from 1.28 or 1.29?

Reply:
will need to update to v1.28 but since it is just a chart sync PR...i will make that change in the new PR where i will update this as well and security-scan tag etc.